std_msgs/Header header
sensor_msgs/Image back_img
sensor_msgs/Image decos_img
jsk_recognition_msgs/BoundingBoxArray deco_bboxes
geometry_msgs/Point bimg_lt_pos
geometry_msgs/Point bimg_rb_pos
geometry_msgs/Point visual_point
---
geometry_msgs/PoseArray result_pos
